src/main/java/org/openstreetmap/atlas/utilities/archive/Extractor.java

```diff
@@ -197,7 +195,7 @@ else if (outputFile.exists() && this.skipExisting)
                 }
                 else if (current.isDirectory())
                 {
-                    continue;
+                    // Continue
```

Comment:
should this be commented or just removed?

Reply:
Well, sonar complained about using continue here since it does not change the behavior. But when I removed it it complained about an empty method body. So here we go :)